### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by Aloso** [Aloso/lexical-sort#1]
Currently, there is no benchmark. Let me know if you want to see how fast this crate is, and I'll write one!

**Comment by ArturKovacs** [comment]
I would be interested in comparing this with `alphanumeric-sort` 😊

**Comment by ArturKovacs** [comment]
Let me rephrase; I'd be interested in seeing a comparison

**Comment by Aloso** [comment]
@ArturKovacs check out the updated README. You can run the benchmarks yourself by executing `cargo bench`, which generates violin graphs under `target/criterion/report/index.html`, if you have Gnuplot installed.
